### PR TITLE
Added Loggng

### PR DESCRIPTION
**Pull request by dcowden**

### CODE REVIEW

File: src/main/java/frc/pathrecognizer/PixyCameraConnector.java [hunk 1]
```diff
@@ -52,6 +54,29 @@ public PixyCameraConnector() {
             block.getAngle()
           );
 
+          switch  (count){
```
Comment (dcowden) [inline src/main/java/frc/pathrecognizer/PixyCameraConnector.java]: too much duplication. there's no need for the count variable.
a simple call to block.getWidth(), block.getHeight() will be enough.
if you'd like to konw how many targets there are without counting them up, just log out the value of blocks.size()

File: src/main/java/frc/pathrecognizer/VisionFieldLayoutRecognizer.java [hunk 2]
```diff
@@ -44,20 +46,23 @@ private double computeAverageDistance(List<FieldVisionInput> fieldVisionInputPar
...
   }
 
   public AutonomousPath detectPose(List<FieldVisionInput> fieldVisionInputParameter) {
         
       AutonomousPath autonomousPath;
-      
       VisionPathName visionPathName = VisionPathName.CouldNotDeterminePath;
        
-      // Just for refernce the formula to calculate the focal length
+      // Just for reference the formula to calculate the focal length
```
Comment (dcowden) [inline src/main/java/frc/pathrecognizer/VisionFieldLayoutRecognizer.java:60]: thank you for fixing this typo!

File: src/main/java/frc/robot/subsystems/PathRecognizerSubsystem.java [hunk 3]
```diff
@@ -26,6 +26,7 @@ public AutonomousPath getAutonomousPath(){
 
         if (pixyCameraConnector.getPixyConnectionStatus()){
             fieldVisionInputList = pixyCameraConnector.getFieldVisionInput();
+
```
Comment (dcowden) [inline src/main/java/frc/robot/subsystems/PathRecognizerSubsystem.java]: dont need this line.

File: src/test/java/frc/pathrecognizer/TestVisionFieldLayoutRecognizer.java [hunk 4]
```diff
@@ -6,6 +6,7 @@
 import static org.junit.Assert.assertEquals;
 
 public class TestVisionFieldLayoutRecognizer {
+
```
Comment (dcowden) [inline src/test/java/frc/pathrecognizer/TestVisionFieldLayoutRecognizer.java]: dont need this blank line

File: src/main/java/frc/pathrecognizer/PixyCameraConnector.java [hunk 5]
```diff
@@ -52,6 +54,29 @@ public PixyCameraConnector() {
...
+          case 2:
+          {
+          logger.log("Pixel Width 2: ", block.getWidth());
+          logger.log("PixelHeight: 2: ", block.getHeight());
+          break;
+          }
+          case 3:
+          {
+          logger.log("Pixel Width 3: ", block.getWidth());
+          logger.log("PixelHeight 3: ", block.getHeight());
```
Comment (dcowden) [inline src/main/java/frc/pathrecognizer/PixyCameraConnector.java]: at this point, alarm bells MUST have been going off! the same code 3 times? please NO

File: src/main/java/frc/pathrecognizer/PixyCameraConnector.java [hunk 6]
```diff
@@ -52,6 +54,29 @@ public PixyCameraConnector() {
...
+          break;
+          }
+        }
+
+          count = count + 1;
```
Comment (dcowden) [inline src/main/java/frc/pathrecognizer/PixyCameraConnector.java]: it should feel wrong when you have a counter variable inside of a foreach loop.

**Comment by rohitgan2004** [comment]
Fixed all of the above sir, will check in the code now.

**Pull request merged**